ironic/tests/unit/drivers/modules/drac/test_management.py

```diff
@@ -821,3 +825,85 @@ def test_clear_job_queue(self, mock_get_drac_client):
                 job_ids=['JID_CLEARALL_FORCE'])
 
             self.assertIsNone(return_value)
+
+
+class DracRedfishManagementTestCase(test_utils.BaseDracTest):
+
+    def setUp(self):
+        super(DracRedfishManagementTestCase, self).setUp()
+        self.config(enabled_hardware_types=['idrac'],
+                    enabled_power_interfaces=['idrac-redfish'],
+                    enabled_management_interfaces=['idrac-redfish'],
+                    enabled_inspect_interfaces=['idrac-redfish'])
+        self.node = obj_utils.create_test_node(
+            self.context, driver='idrac',
+            driver_info=DRAC_REDFISH_INFO_DICT)
+
+    def test_get_properties(self):
+        with task_manager.acquire(self.context, self.node.uuid,
+                                  shared=True) as task:
+            properties = task.driver.get_properties()
+            for prop in redfish_utils.COMMON_PROPERTIES:
+                self.assertIn(prop, properties)
+
+    @mock.patch.object(requests, 'get', autospec=True)
+    @mock.patch.object(requests, 'delete', autospec=True)
+    @mock.patch.object(redfish_utils, 'get_system', autospec=True)
+    def test_clear_job_queue(self, mock_get_system, mock_delete, mock_get):
+        fake_system = mock.Mock()
+        mock_get_system.return_value = fake_system
+        with task_manager.acquire(self.context, self.node.uuid,
+                                  shared=False) as task:
+            task.node.driver_info['redfish_address'] = '1.2.3.4'
+            return_value = task.driver.management.clear_job_queue(task)
+            fake_system.clear_job_queue(task)
```

Comment:
I don't think the line above does anything because you're just calling clear_job_queue on a mock.  It should probably be removed.

Reply:
Addressed.